### ISSUE TITLE
Quick Links should be underlined

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/DSS#359]
### Steps to recreate
1. Install axe devtools if you don't have it already
2. Go to https://dss.princeton.edu/catalog/
3. Scan the page using axe devtools.
4. Note that you get 7 instances of "Links must be distinguishable without relying on color"

### URL

https://dss.princeton.edu/catalog/

### Issues

Links are only different from the surrounding text by color.

### Severity

Dubbot and axe devtools rank this as serious.

### WCAG Conformance level (A, AA, AAA)

A

### User impact

It's hard to distinguish links from surrounding text, which can make the "quick" links much less quick to work with.

### Recommended fix

Add an underline to these links

### Acceptance criteria
- [x] I can successfully use these links with Voice Over in Safari.
- [x] I have checked this feature using axe devtools, and confirmed that it resolves the 7 instances of "Links must be distinguishable without relying on color" while not adding any additional level A or AA issues.

### Notes

Dubbot page: https://princeton.dubbot.com/sites/637d382d6f2342000118da27/pages/637d38361da265fd22b17fd7?share_token=page-dUQvYTZiQXhjcXJuTEVoYlRHMG52RlpOZWJGckhPSzY4aXdHZk1MNnE1NnBFdnR4T3hIR0JWSUdVWHBGT1h1aC0tYU44SlhLaDNUaG1uQXV3V05aWFc3UT09--e74c9e4924e702556885bb02e3afc08a6a0e84df
